src/server/pfs/fuse/files_linux.go

```diff
@@ -31,3 +32,13 @@ func (f *loopbackFile) utimens(a *time.Time, m *time.Time) syscall.Errno {
 	err := futimens(int(f.fd), &ts)
 	return fs.ToErrno(err)
 }
+
+// futimens - futimens(3) calls utimensat(2) with "pathname" set to null and
+// "flags" set to zero
+func futimens(fd int, times *[2]syscall.Timespec) (err error) {
```

Comment:
Named return params like this have bitten me a few times with shadowing so we generally only do it when you need to access the parameter in a deferred function. Could you switch this to just returning the error?

Reply:
Just realized this is copied from elsewhere so actually I was the one who checked this in originally >.< . (I was just copying it from the fuse library.) Given that I'm cool with leaving it as is.